openassessment/xblock/static/js/spec/lms/oa_staff_area.js

```diff
@@ -354,12 +354,13 @@ describe('OpenAssessment.StaffAreaView', function() {
         it('shows the correct buttons for full grading', function() {
             var view = createStaffArea({}, 'oa_staff_area_full_grading.html'),
                 $buttons = $('.ui-staff__button', view.element);
-            expect($buttons.length).toBe(4);
+            expect($buttons.length).toBe(5);
             expect($buttons).toHaveAttr('aria-expanded', 'false');
             expect($($buttons[0]).text().trim()).toEqual('Manage Individual Learners');
             expect($($buttons[1]).text().trim()).toEqual('View Assignment Statistics');
             expect($($buttons[2]).text().trim()).toEqual('Grade Available Responses');
-            expect($($buttons[3]).text().trim()).toEqual('View ORA in Studio');
+            expect($($buttons[3]).text().trim()).toEqual('Demo the new Grading Experience');
+            expect($($buttons[4]).text().trim()).toEqual('View ORA in Studio');
```

Comment:
Are we missing the negative version of this test? When ESG is enabled, should not show the Demo/Grade buttons.